### DISCUSSION TITLE
AVG Watts per Lap

### DISCUSSION THREAD

**Discussion by fuzzykeys** [cagnulein/qdomyos-zwift#1009] [Ideas]
It would be great if we could either see our AVG Watts/lap. Ideally, we'd have the option to add a new tile to the QZ screen, but resetting the existing AVG Watt tile when we start a new lap could also be helpful if that is not feasible. Thanks!

**Comment by cagnulein** [comment]
Yes I guess the best solution is to add a new tile called "avg watt lap"
I will try to do in the next days!
